app/models/user.rb

```diff
@@ -127,6 +127,16 @@ def twitter
     get_authorization(:twitter)
   end
 
+  # Returns true when this user has a twitter authorization
+  def github?
+    has_authorization?(:github)
+  end
+
+  # Returns the twitter authorization
```

Comment:
doc not in sync with method. maybe call these methods `github_authorization` and `github_authorization?`